Dockerfile

```diff
@@ -22,14 +22,16 @@ ENV NB_UID=jovyan
 ENV FOLDER=netpyne
 ARG GEPPETTO_VERSION=development
 ARG BUILD_ARGS=""
+ARG NETPYNE_VERSION=master
+ARG WORKSPACE_VERSION=master
 
 ENV FOLDER=/home/jovyan/work/NetPyNE-UI
 
 USER root
 
 RUN rm -rf /var/lib/apt/lists
 RUN apt-get update -qq &&\
-    apt-get install python3-tk vim nano unzip git make libtool g++ -qq pkg-config libfreetype6-dev libpng-dev libopenmpi-dev -y
+    apt-get install python3-tk vim nano unzip git make libtool g++ -qq pkg-config libfreetype6-dev libpng-dev libopenmpi-dev  openjdk-11-jre-headless -y -y
```

Comment:
Do we need the open jdk? I'm unsure we use it or not

Reply:
@aranega JDK is rquired as some of the NeuroML->NetPyNE pipeline is built on Java and needs to execute in a separate process